rest/data/host_create.go

```diff
@@ -259,71 +255,9 @@ func createHostFromCommand(cmd model.PluginCommandConf) (*apimodels.CreateHost,
 
 // MakeHost creates a host or container to run for host.create.
 func MakeHost(ctx context.Context, env evergreen.Environment, taskID, userID, publicKey string, createHost apimodels.CreateHost, distro distro.Distro) (*host.Host, error) {
```

Comment:
This function isn't really needed anymore since it's just a direct wrapper of makeEC2IntentHost, we can just make that public and call it directtly.